gio/Gir.toml

```diff
@@ -1333,6 +1332,15 @@ manual_traits = ["SocketControlMessageExtManual"]
     manual = true
     doc_trait_name = "SocketControlMessageExtManual"
 
+[[object]]
+name = "Gio.SocketListener"
+status = "generate"
+manual_traits = ["SocketListenerExtManual"]
+    [[object.function]]
+    name = "incoming"
```

Comment:
Is this function does not exist in C, you don't have to list it here :) It has no effect

Reply:
I see. So just the `manual_traits` bit is what's important?

Reply:
Not sure how this came back

Reply:
:/ I think I switched computers and started working off of the wrong base